qemu/tests/cfg/cdrom_test.cfg

```diff
@@ -56,15 +56,15 @@
         remove_file_cmd = "del /f /q ${tmp_dir}%s"
         copy_file_cmd = "copy /y %s\%s ${tmp_dir}\"
         tray_check_cmd = "c:\python27\python.exe ${tray_check_src} %s"
-        lock_cdrom_cmd = "D:\eject.exe -i on %s"
-        unlock_cdrom_cmd = "D:\eject.exe -i off %s"
-        eject_cdrom_cmd = "D:\eject.exe %s"
-        close_cdrom_cmd = "D:\eject.exe -t %s"
+        lock_cdrom_cmd = "eject.exe -i on %s"
```

Comment:
it looks it should be eject not eject.exe? 
can you provider your test results?